google/cloud/pubsub/internal/default_batch_sink_test.cc

```diff
@@ -135,6 +139,7 @@ TEST(DefaultBatchSinkTest, BasicWithCompression) {
       std::move(mock), background.cq(),
       DefaultPublisherOptions(
           pubsub_testing::MakeTestOptions()
+              .set<GrpcCompressionAlgorithmOption>(GRPC_COMPRESS_NONE)
```

Comment:
@alevenberg maybe we should consider `using pubsub::CompressionAlgorithmOption = google::cloud::GrpcCompressionAlgorithmOption;`

Reply:
That might be problematic, given that `pubsub::CompressionAlgorithmOption` is given a default value (of `GRPC_COMPRESS_DEFLATE`).